docs/template_plugin/tests/functional/op_reference/mvn.cpp

```diff
@@ -0,0 +1,268 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <ie_core.hpp>
+#include <ie_ngraph_utils.hpp>
+#include <ngraph/ngraph.hpp>
+#include <shared_test_classes/base/layer_test_utils.hpp>
+#include <tuple>
+
+#include "base_reference_test.hpp"
+
+using namespace ngraph;
+using namespace InferenceEngine;
+
+struct Tensor {
```

Comment:
I propose to wait for PR https://github.com/openvinotoolkit/openvino/pull/6664 because looks like it should be a common part

Reply:
ok, I will wait for this PR.

Reply:
Tensor is replaced with version from PR #6664.